models/engine/file_storage.py

```diff
@@ -68,3 +69,17 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """returns the object based on the class and its ID, or None if not found"""
```

Comment:
Please double-check your pycodestyle, you have 5 characters more than allowed

Reply:
ok it was solved